Formula/s/spqr.rb

```diff
@@ -0,0 +1,16 @@
+class Spqr < Formula
+  desc "Project for creating Go projects using hexagonal architecture"
+  homepage "https://github.com/spqr-go/spqr"
+  url "https://github.com/spqr-go/spqr/releases/download/spqr/spqr"
+  version "0.1.0"
+  sha256 "4848df26b2c33d6f263ecbe841cc8b0e627537b0c281abb3fc2235e6125c7704"
+  license "MIT"
+
+  def install
+    bin.install "spqr"
+  end
+
+  test do
+    system "#{bin}/spqr", "help"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.